contracts/Organized.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.4.23;
+
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "./OrganizationInterface.sol";
+
+
+/**
+ * @title Organized contract.
+ *
+ * @notice The Organized contract facilitates integration of
+ *         organization administration keys with different contracts like
+ *         TokenRules, CustomRules etc. Entity has the
```

Comment:
Suggest striking specific names of other contracts in support of future-proofing this documentation.